### PR TITLE
fix mobile styling

### PR DESCRIPTION
**Pull request by chudeki8**
Fixes #53 and some of #26 
- added the meta viewport tag to any pages that didn't have it
- corrected max-width CSS for elements so they don't jump out of their parent container
- little media queries for banner + top-bar spacing

### CODE REVIEW

File: stylesheets/style.css [hunk 1]
```diff
@@ -132,7 +119,7 @@ table tbody tr:nth-of-type( even ) {
 }
 
 tr{
-  
+
 }
```
Comment (barrucadu) [inline stylesheets/style.css]: Could this `tr` just be deleted entirely? The `td` above was.

File: stylesheets/style.css [hunk 2]
```diff
@@ -1812,5 +1799,30 @@ table.fileboard .intro a {
...
+}
 
+/* boardlist gets all scrunched on small screens */
+@media screen and (max-width: 600px) {
+    header, img.board_image {
+        margin-top: 5em;
+    }
+}
+
+@media screen and (min-width: 700px) {
```
Comment (barrucadu) [inline stylesheets/style.css]: I admit, I'm not much of a front-end guy, but is it intentional that this is `min-width: 700px` whereas the prior rule is `max-width: 600px`? What happens if the width is `650px`?

**Comment by barrucadu** [comment]
As a general comment, you should try to avoid needless whitespace changes, as they just clutter the diff. Although, the whitespace is really bad in lainchan's HTML, for some reason...

**Comment by mkwia** [comment]
Desktop styling needs a fix too #52

**Comment by Appleman1234** [comment]
This pull request has conflicts and has already been partially merged in existing commits. Closing.
Feel free to reopen or make a new pull request for specific mobile styling.

**Pull request closed without merging**